.changeset/hungry-crews-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+"miniflare": patch
+---
+
+fix(miniflare): api proxy preserve multiple Set-Cookie headers
```

Comment:
Since this is a miniflare specific fix including "miniflare" in the changelog itself is not necessary :slightly_smiling_face: 

```suggestion
fix: api proxy preserve multiple Set-Cookie headers
```

Reply:
updated and rebased, thanks!